b2b-sample/guardio-insurance-sample-application/components/settingsComponents/application/confirmAddRemoveLoginFlowModal.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import { Avatar, Button, Col, Grid, Loader, Modal, Row, useToaster } from 'rsuite';
+import stylesSettings from '../../../styles/Settings.module.css';
+import decodePatchApplicationAuthSteps from
+    '../../../util/apiDecode/settings/application/decodePatchApplicationAuthSteps';
+import { PatchApplicationAuthMethod } from '../../../util/util/applicationUtil/applicationUtil';
+import { checkIfJSONisEmpty } from '../../../util/util/common/common';
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from '../../../util/util/frontendUtil/frontendUtil';
+import { errorTypeDialog, successTypeDialog } from '../../util/dialog';
+
+export default function ConfirmAddRemoveLoginFlowModal(props) {
+
+    const toaster = useToaster();
+
+    const [loadingDisplay, setLoadingDisplay] = useState(LOADING_DISPLAY_NONE);
+
+    const onSubmit = async (patchApplicationAuthMethod) => {
+        setLoadingDisplay(LOADING_DISPLAY_BLOCK);
+
+        decodePatchApplicationAuthSteps(props.session, props.applicationDetail, props.idpDetails,
+            patchApplicationAuthMethod)
+            .then((response) => props.idpIsinAuthSequence
+                ? onIdpRemovefromLoginFlow(response)
+                : onIdpAddToLoginFlow(response))
+            .finally((response) => setLoadingDisplay(LOADING_DISPLAY_NONE));
+    }
+
+    const onRemove = async () => {
+        await onSubmit(PatchApplicationAuthMethod.REMOVE);
+    }
+
+    const onAdd = async () => {
+        await onSubmit(PatchApplicationAuthMethod.ADD);
+    }
+
+    const onSuccess = () => {
+        props.onModalClose();
+        props.fetchAllIdPs().finally();
+    }
+
+    const onIdpAddToLoginFlow = (response) => {
+        if (response) {
+            onSuccess();
+            successTypeDialog(toaster, "Success", "Identity Provider Add to the Login Flow Successfully.");
+        } else {
+            errorTypeDialog(toaster, "Error Occured", "Error occured while adding the the identity provider.");
+        }
+    }
+
+    const onIdpRemovefromLoginFlow = (response) => {
+        if (response) {
+            onSuccess();
+            successTypeDialog(toaster, "Success", "IIdentity Provider Remove from the Login Flow Successfully.");
```

Comment:
Please check spelling and grammer

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/3256f90f042ce73fd772125bb148f575e939ca09